spec/src/main/asciidoc/ch03-entity-operations.adoc

```diff
@@ -1844,112 +1844,104 @@ it to be applied to that entity or mapped superclass and its subclasses.
 In the case of multiple callback methods for
 a single lifecycle event, the invocation order rules described in <<a2251>> apply.
 
-=== Bean Validation [[a2366]]
+=== Jakarta Validation [[a2366]]
 
-This specification defines support for use of
-Bean Validation <<a19498>> within Jakarta Persistence
-applications.
+Jakarta Persistence integrates with Jakarta Validation <<a19498>>
+and specifies additional validation points for managed types
+(i.e. entities, mapped superclasses, and embeddable classes)
+at the entity lifecycle events described in <<a2202>>.
 
-Managed classes (entities, mapped
-superclasses, and embeddable classes) may be configured to include Bean
-Validation constraints.
-
-Automatic validation using these constraints
-is achieved by specifying that Jakarta Persistence delegate validation to
-the Bean Validation implementation upon the pre-persist, pre-update, and
-pre-remove entity lifecycle events described in <<a2202>>.
-
-Validation can also be achieved by the
-application calling the `validate` method of a `Validator` instance upon
-an instance of a managed class, as described in the Bean Validation
-specification <<a19498>>.
+Jakarta Persistence provider may automatically trigger validation of the managed objects
+as defined by the Jakarta Validation specification <<a19498>>.
 
 ==== Automatic Validation Upon Lifecycle Events
 
-This specification supports the use of bean
-validation for the automatic validation of entities upon the
-`pre-persist`, `pre-update`, and `pre-remove` lifecycle validation events.
-These lifecycle validation events occur immediately after the point at
-which all the `PrePersist`, `PreUpdate`, and `PreRemove` lifecycle
-callback method invocations respectively have been completed, or
-immediately after the point at which such lifecycle callback methods
-would have been completed (in the event that such callback methods are
-not present).
+If enabled, the automatic validation of entities
+upon lifecycle validation events must occur immediately after the point at
+which all the corresponding lifecycle callback method invocations have been completed,
+or immediately after the point at which such lifecycle callback methods
+would have been completed (if such callback methods are absent).
 
 [NOTE]
 ====
-In the case where an entity is persisted and
-subsequently modified in a single transaction or when an entity is
-modified and subsequently removed in a single transaction, it is
-implementation dependent as to whether the pre-update validation event
-occurs. Portable applications should not rely on this behavior.
+In the same way as with entity lifecycle events (<<a2202>>),
+the number of times automatic validation is triggered
+for an entity within a single transaction,
+when multiple operations are performed on the same entity,
+depends on the Jakarta Persistence provider.
```

Comment:
I had to read this sentence three times to understand what it was saying. Think of a different way to structure it. (Also, "In the same way as with" is inelegant.)